### DISCUSSION TITLE
Override Fluent default styling

### DISCUSSION THREAD

**Discussion by SirZeeno** [amwx/FluentAvalonia#554] [Q&A] [answered]
I am trying to use this styling below to make the button itself invisible but the text within light up when hovered over. But with the switch over to FluentAvaloniaUI it stopped working as intended. So my question is, how do I override the default styling to allow me to style my button as I want again?

        
        <Style Selector="Button.transparentButton">
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="BorderBrush" Value="Transparent"/>
            
            <Setter Property="Foreground" Value="#929292"/>
        </Style>
        
        
        <Style Selector="Button.transparentButton:pointerover /template/ ContentPresenter#PART_ContentPresenter">
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="BorderBrush" Value="Transparent"/>
            <Setter Property="Foreground" Value="#96b6d0"/>
        </Style>
        
        <Style Selector="Button.transparentIcon:pointerover /template/ ContentPresenter#PART_ContentPresenter">
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="BorderBrush" Value="Transparent"/>
        </Style>
        
        
        <Style Selector="Button.transparentIcon Image">
            <Setter Property="Opacity" Value="0.5"/> 
        </Style>
        
        
        <Style Selector="Button.transparentIcon:pointerover Image">
            <Setter Property="Opacity" Value="1"/> 
        </Style>

**Comment by timunie** [comment]
lookup original styles which you can explore here on GH

**Reply by SirZeeno** [reply]
Sorry, I am not entirely sure what you are referring to. A lot of what I am finding when searching for "styles" in the repository is hard-coded setters and one discussion about button styles which doesn't help me much. Could you please explain what exactly you mean and how those styles could help me fix my issue?

**Reply by timunie** [reply, marked as answer]
https://github.com/amwx/FluentAvalonia/tree/master/src/FluentAvalonia/Styling/ControlThemes 

read the selectors used there, to understand which ones you may need.

**Reply by SirZeeno** [reply]
Thank you! Thats exactly what ive been looking for.
